src/DynamoCore/Graph/Presets/PresetModel.cs

```diff
@@ -10,6 +10,7 @@ namespace Dynamo.Graph.Presets
     /// This class references a set of nodemodels, and a set of serialized versions of those nodemodels
     /// a client can use this class to store the state of a set of nodes from a graph
     /// </summary>
+    [Obsolete("Presets are not used anymore", false)]
```

Comment:
Can you align the obsolete message with other examples in DynamoCore, e.g. `Property/ Class is obsolete in DynamoCore` or `Model/ Class is removed from DynamoCore`, etc